sdks/python/apache_beam/coders/coders.py

```diff
@@ -190,7 +190,7 @@ def _get_component_coders(self):
     # refined in user-defined Coders.
     return []
 
-  def as_cloud_object(self):
+  def as_cloud_object(self, coders_context=None):
```

Comment:
Is there any reason to make coders_context optional?

Reply:
I was striving to avoid changing the behavior of this code for anyone other than the dataflow runner, and I'm not sure what other code that might be.  There also are a fair number of current tests that don't need to pass this context.  This is how I did the Java version of this change, too.

Reply:
Cloud objects are exclusively a dataflow thing. I was thinking this might be helpful ensuring it's passed everywhere that it's required. Being minimally invasive and not having to change tests is a fine reason to make this optional.